src/modules/contract/contract.service.spec.ts

```
@@ -0,0 +1,108 @@
+import { Clause } from '../../entities/clause.entity';
+import { RiskFlag } from '../../entities/risk-flag.entity';
+import { Summary } from '../../entities/summary.entity';
+import { QnA } from '../../entities/qna.entity';
+import { HumanReview } from '../../entities/human-review.entity';
+import { AiService } from '../ai/ai.service';
+import { NotFoundException, BadRequestException } from '@nestjs/common';
+
+describe('ContractService', () => {
+  let service: ContractService;
+  let contractRepository: jest.Mocked<Repository<Contract>>;
+  let clauseRepository: jest.Mocked<Repository<Clause>>;
```

ESLint

[error] 'clauseRepository' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)